tutorials/05_starting_ray/05_starting_ray_builtin.ipynb

```diff
@@ -5,23 +5,25 @@
    "id": "4cba3240",
    "metadata": {},
```

Comment:
ensure all the metadata for the kernelspec and interpreter are taken out of each notebook. (I usually do this by just manually editing the ipynb file in vim)

Reply:
Fixed in c0fb85c